management/storageservice/entities.go

```diff
@@ -33,19 +33,41 @@ type StorageServiceProperties struct {
 	GeoPrimaryRegion      string
 }
 
-type StorageServiceDeployment struct {
-	XMLName               xml.Name `xml:"CreateStorageServiceInput"`
-	Xmlns                 string   `xml:"xmlns,attr"`
-	ServiceName           string
-	Description           string
-	Label                 string
-	AffinityGroup         string `xml:",omitempty"`
-	Location              string `xml:",omitempty"`
-	GeoReplicationEnabled bool
-	ExtendedProperties    ExtendedPropertyList
-	SecondaryReadEnabled  bool
+type CreateStorageServiceInput struct {
+	XMLName xml.Name `xml:"http://schemas.microsoft.com/windowsazure CreateStorageServiceInput"`
+	StorageAccountCreateParameters
+}
+
+type StorageAccountCreateParameters struct {
+	// Required. A name for the storage account that is unique within Azure. Storage account names must be
+	// between 3 and 24 characters in length and use numbers and lower-case letters only.
+	// This name is the DNS prefix name and can be used to access blobs, queues, and tables in the storage account.
+	ServiceName string
+	// Optional. A description for the storage account. The description may be up to 1024 characters in length.
+	Description string `xml:",omitempty"`
+	// Required. A label for the storage account specified as a base64-encoded string. The label may be up to
+	// 100 characters in length. The label can be used identify the storage account for your tracking purposes.
```

Comment:
I think comments are unnecessary. You're duplicating the RDFE contract here. Unless you auto-generate the code from the spec, if the requirement tomorrow becomes 90 chars max, your comment is out of sync and you can't just manually maintain that. Let's clear these commands.